ansible/roles/common/tasks/main.yml

```diff
@@ -74,7 +74,7 @@
 
 - name: Add apt repository for stable version
   apt_repository:
-    repo: deb [arch=arm64] https://download.docker.com/linux/ubuntu xenial stable
+    repo: "deb [arch=arm64] https://download.docker.com/linux/ubuntu {{ ansible_lsb.codename}} stable"
```

Comment:
This is really cool, would `{{ ansible_lsb.id|lower }} ` make more sense than `{{ ansible_lsb.codename}}` ?

Reply:
`codename` appears to be the best option. Nice work!

```
        "ansible_lsb": {                                                                                               
            "codename": "precise",                                                                                     
            "description": "Ubuntu 12.04 LTS",                                                                         
            "id": "Ubuntu",                                                                                            
            "major_release": "12",                                                                                     
            "release": "12.04"                                                                                         
        }, 
```